packages/babel-core/test/config-chain.js

```diff
@@ -51,6 +53,31 @@ function loadOptionsAsync(opts) {
   return babel.loadOptionsAsync({ cwd: __dirname, ...opts });
 }
 
+async function loadOptionsAsyncInSpawedProcess({ filename, cwd }) {
+  // !!!! hack is coming !!!!
+  // remove this section when https://github.com/facebook/jest/issues/9430 is resolved
```

Comment:
Nit: add a `TODO(Babel 8)` here. Hopefully we'll be able to update jest to a working version for Babel 8, and we'll catch this comment when checking if there is anything left to do for the major release. Worst case we'll defer removing this 😬